infrastructure/terraform/environment/sbox-hmi.tfvars

```diff
@@ -5,7 +5,7 @@ publisher_email = "sandbox@hmcts.placeholder"
 protocols = ["http", "https"]
 revision = "1"
 open_api_spec_content_format = "swagger-link-json"
-open_api_spec_content_value = "https://raw.githubusercontent.com/hmcts/reform-api-docs/master/docs/specs/fh-request-hearing.json"
+open_api_spec_content_value = "https://raw.githubusercontent.com/hmcts/reform-api-docs/master/docs/specs/future-hearings-request-hearing.json"
```

Comment:
The file is generic and does not belong to request hearing, I have changed this and pushed a new commit